index.html

```diff
@@ -125,40 +132,40 @@ <h2 class="title">Tonic</h2>
                     </div>                
             </article>
 <!--card_4 starts from here-->
-            <article class="grid_container">
-                <img id="picture4" class="grid_elements" src="./assets/pictures/picture4.jpg" alt=""/>
-                    <div class="grid_elements">                       
-                        <h2 class="title">Multi-Post<br>Stories</h2>
-                            <div class="canopy_line">
+            <article class="grid_container" id="reverse">
+                <img class="grid_elements" src="./assets/pictures/picture4.jpg" alt=""/>
+                    <div class="grid_elements" id="reverse_elements">                       
+                        <h2 class="title1_r">Tonic</h2>
+                            <div class="canopy_line_r">
                                 <a class="canopy">CANOPY</a>   
                                 <a class="bull">&bull;</a>  
                                 <a class="bull">Back End Dev</a>  
                                 <a class="bull" >&bull;</a>  
                                 <a class="bull">2022</a>                  
                             </div>
-                        <p class="content">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                        <div class="program">
+                        <p class="content_r">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <div class="program_r">
                             <button type="button" class="pro">html</button>
                             <button type="button" class="pro">css</button>
                             <button type="button" class="pro">javascript</button>                               
                         </div>
                         <button type="button" class="see_button">See Project</button>
                     </div>                
-            </article>                     
+            </article>
 <!--about myself section from here-->
-            <section id="myself_div">    
-                <div class="picture_container">
+            <section class="about">    
```

Comment:
Please fix this issue with the about section as requested by previous reviewer.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/171985140-991d07e7-0c29-4e3a-b9ed-4a758ab84307.png) | ![image](https://user-images.githubusercontent.com/33268927/171985168-5d569b07-da65-4621-9f63-8c8da5fcdb85.png) |